datarepo-clienttests/src/main/java/scripts/utils/DataRepoUtils.java

```diff
@@ -37,7 +37,8 @@
 
   private DataRepoUtils() {}
 
-  private static int maximumSecondsToWaitForJob = 500;
+  private static int maximumSecondsToWaitForJob =
+      Long.valueOf(TimeUnit.HOURS.toSeconds(2)).intValue();
```

Comment:
Is this preferable to writing
```suggestion
      (int) TimeUnit.HOURS.toSeconds(2);
```